src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const employees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (salary) => {
+  return salary.replace(/\D/g, '');
+};
+
+const sortList = (list) => {
+  list
+    .toSorted(
```

Comment:
There is no method 'toSorted' in JavaScript. You probably meant to use 'sort' method.